src/assets/server/rest/v1/schemas/auth/auth-eula.json

```diff
@@ -0,0 +1,12 @@
+{
+  "title": "Eula Request",
+  "type": "object",
+  "properties": {
+    "Language": {
+      "$ref": "common.json#/definitions/language"
+    },
+    "tenant": {
```

Comment:
Why do we have the param tenant's first letter in lower case here and not the language?

Reply:
I followed the way AuthSecurtiy gather informations
`Language` param starts with an uppercase letter because it comes from Query, and `tenant` param comes from the header of the request. I guess there is a technical choice behind this ? Otherwise, I could just move tenant param to Query, tell me

Reply:
@jerome-benoit Could you have a look?

Reply:
> @jerome-benoit Could you have a look?

If it's a POST request on an endpoint, all data must come in the body or the headers.
If it's a GET request on an endpoint, query string should be avoided in favor of URI with parameters parsed: they then can be case sensitive, unlike in the query string part, among one of numerous advantages. 
restapi.com/authors/:id -> author with id
restapi.com/authors/:name -> authors with name
restapi.com/authors/:town -> authors in the town
restapi.com/authors/:id/town -> authors town with id
etc.

Reply:
If it's related to the GET Eula, the tenant is deduced from the URL and should be passed in the Query string and named Tenant.

Reply:
Done

Reply:
> If it's related to the GET Eula, the tenant is deduced from the URL and should be passed in the Query string and named Tenant.

Query string are case insensitive. A more correct GET URI is /eula/.../:tenantID

The new REST API shall never ever use query string, only well constructed URIs with parameters inside, each of them matching an express router. That's the whole purpose of having made a hierarchical express routers tree: being able to build meaningful URI to query in a REST standardised manner.

Reply:
Just realized that the EULA is not tenant relevant, so why are we talking about passing it in the end :) !?
Only the Language is relevant here and can be a query string as this is not an identifier.

Reply:
Therefore, should we only use eulas stored in the default tenant ? Currently, each tenant has its own `eula` collection, that's why we are using tenant identifier here

Reply:
Done, removed tenant related parameters/attributes